ansible/roles/kafka-logging-setup/files/kafka-cluster-persistent.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+  namespace: "amq"
+spec:
+  entityOperator:
+    topicOperator:
+      reconciliationIntervalSeconds: 90
+    userOperator:
+      reconciliationIntervalSeconds: 120
+  kafka:
+    config:
+      log.message.format.version: "2.5"
+      offsets.topic.replication.factor: 3
+      transaction.state.log.min.isr: 2
+      transaction.state.log.replication.factor: 3
+      message.max.bytes: 10485760
+    listeners:
+      plain:
+        authentiation:
+          type: scram-sha-512
+      tls:
+        authentiation:
+          type: tls
+    replicas: 3
+    NodeSelector:
+      node-role.kubernetes.io/infra: ""
```

Comment:
I think we should make the node selectors in this file configurable so when we lay down local storage and want the kafka nodes separate from the prometheus nodes we can use a different node selector.